src/nav-bar.js

```diff
@@ -2,36 +2,35 @@ import React from 'react';
 
 class NavBar extends React.Component {
     componentDidMount(){
-        /*
-        var fixedContainer = document.getElementById("fixed-container");
+        var fixedContainer = document.getElementById("navigation-container");
         var children = Array.from(fixedContainer.children);
 
         children.forEach(child => {
             var clone = child.cloneNode(true);
             clone.removeAttribute("id");
             clone.style.visibility = "hidden";
-
+            clone.classList.add("desktop-hide");
```

Comment:
I'd prefer to do this in CSS but we can address this in the layout refactor

Reply:
What it's doing is duplicating HTML & adding a class to the clones, I can further explain what this is for at some point & see if you think there's a better way to do it

Reply:
I understand what it's doing and I think there's a much better way -- this code style is repeated already though, so I think it would be clearer to refactor it all at once (next PR) vs. try to refactor just this change now

Reply:
It's okay for now!